scripts/build/termux_step_replace_guess_scripts.sh

```diff
@@ -4,4 +4,11 @@ termux_step_replace_guess_scripts() {
 	cd "$TERMUX_PKG_SRCDIR"
 	find . -name config.sub -exec chmod u+w '{}' \; -exec cp "$TERMUX_SCRIPTDIR/scripts/config.sub" '{}' \;
 	find . -name config.guess -exec chmod u+w '{}' \; -exec cp "$TERMUX_SCRIPTDIR/scripts/config.guess" '{}' \;
+
+	if [ "$TERMUX_ON_DEVICE_BUILD" = "true" ] && [ "$TERMUX_PACKAGE_LIBRARY" = "glibc" ]; then
+		local list_files=$(grep -s -r -l '^#!.*/bin/' $TERMUX_PKG_SRCDIR)
+		if [ -n "$list_files" ]; then
+			sed -i "s|#\!.*/bin/|#\!$TERMUX_PREFIX_CLASSICAL/bin/|" $list_files
```

Comment:
> Which glibc specific files does this fix?

This algorithm fixes scripts that could theoretically run at compile time. For understanding, at the time of compiling something based on the glibc library, the value of `LD_PRELOAD` is removed. This is necessary because the bionic-based library is specified there and because of this, a conflict occurs during the launch of glibc commands.